modules/rostests/apitests/shell32/menu.cpp

```diff
@@ -14,6 +14,8 @@
 #define test_S_OK(hres, message) ok(hres == S_OK, "%s (0x%lx instead of S_OK)\n",message, hResult);
 #define test_HRES(hres, hresExpected, message) ok(hres == hresExpected, "%s (0x%lx instead of 0x%lx)\n",message, hResult,hresExpected);
 
+BOOL g_IsVista = GetNTVersion() == _WIN32_WINNT_VISTA;
```

Comment:
```suggestion
BOOL g_IsVista = (GetNTVersion() == _WIN32_WINNT_VISTA);
```
NOTE: This is redundant wrt. the `static const BOOL g_bVista = (GetNTVersion() == _WIN32_WINNT_VISTA);` in `modules/rostests/apitests/shell32/CShellDesktop.cpp`.
Would it be worth having this global defined in one single place in the apitests/shell32, and then being able to use it in the different places in shell32_apitest with some `extern BOOL g_bVista;` or something alike?